dts/bindings/counter/nxp,lptmr.yaml

```
@@ -20,7 +20,13 @@ properties:
   clk-source:
     type: int
     required: true
-    description: Prescaler clock source (0 to 3)
+    description: |
+      Prescaler clock source (0 to 3), represents
+      the input clock source that the lptmr runs on
+      0 <- 12MHz
+      1 <- 16KHz
+      2 <- 32KHz
+      3 <- Output of OSC_SYS Clock
```

```suggestion
      Selects the clock to be used by the LPTMR prescaler/glitch filter.
      In time counter mode, this field selects the input clock to the prescaler.
      In pulse counter mode, this field selects the input clock to the glitch filter.
      The clock connections vary by device, see the device reference manual for
      more details.
```

We shouldn't add SoC specific clock values to a generic hw IP spec. Alternatively, if you need to have this described in the binding for some reason, you can create an `nxp,<your-soc>-lptmr.yaml` binding that inherits from this binding and modify the description to match your SoC integration, while keeping the semantics of what this property means.
(context: I'm enabling this peripheral for S32K1)

---

Thank you for the comment, sorry I did not consider this.